packages/broker/configs/docker-1.env.json

```diff
@@ -5,17 +5,29 @@
             "privateKey": "0xaa7a3b3bb9b4a662e756e978ad8c6464412e7eef1b871f19e5120d4747bce966"
         },
         "network": {
-            "id": "0xde1112f631486CfC759A50196853011528bC5FA0",
-            "trackers": {
-                "contractAddress": "0xBFCF120a8fD17670536f1B27D9737B775b2FD4CF"
-            },
-            "location": {
-                "latitude": 60.19,
-                "longitude": 24.95,
-                "country": "Finland",
-                "city": "Helsinki"
+            "layer0": {
+                "peerDescriptor": {
+                    "id": "entryPointBroker",
+                    "type": 0,
+                    "websocket": {
+                        "ip": "127.0.0.1",
+                        "port": 40401
+                    }
+                },
+                "entryPoints": [{
```

Comment:
The development-n.json and docker-n.json seem to very similar after these changes? Maybe we don't need both files? (If so, could do that change in a separate PR)

Reply:
Perhaps, the key difference here is in the Cassandra configuration. The dev env uses `127.0.01` to cconnect to Cassandra where as the Docker env uses the `cassandra` domain name

Reply:
@harbu Do you know what that `cassandra` host means? Maybe we could use 10.200.10.1:9042 in both configs? (See https://github.com/streamr-dev/streamr-docker-dev/blob/95c52ecc0d803e7c0bbaa73708a59ba5babc1cc3/docker-compose.yml#L190)

Not related to this PR. I'll open a separate PR if we can unify the configs this way.